tasks.py

```diff
@@ -0,0 +1,8 @@
+from invoke import task
+
+
+@task
+def run(c):
+    # c.run("python3 ./manage.py makemigrations")
```

Comment:
Need to wait for DB port, then make migrations, then migrate, the collect static